invenio_rdm_records/resources/serializers/iiifp/schema.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 data-futures.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""IIIF Presentation API Schema for Invenio RDM Records."""
+
+from flask import current_app
+from marshmallow import Schema, fields
+
+
+class IIIFPresiSchema(Schema):
+    """IIIF Presentation API Marshmallow Schema."""
+
+    class Meta:  # todo: sort ordering this "should" be first.
+        """Marshmallow meta class."""
+
+        include = {
+           '@context': fields.Constant(
+              'http://iiif.io/api/presentation/2/context.json'
+           ),
+           '@type': fields.Constant("sc:Manifest"),
+           '@id': fields.Method('make_manifest_id')
+        }
+
+    label = fields.String(attribute='metadata.title')
+    metadata = fields.Method('make_metadata')
+    description = fields.Function(
```

Comment:
Oops....
Spec-wise we *should* have a description, its absence can confuse viewers. Suggest it to insert something generic if description not defined?

Reply:
fixed in #b6e84a7

Reply:
It could be done by adding a default also to marshmallow Str:
```
description = fields.Str(attrbute="metadata.description", default="Manifest generated by InvenioRDM")
```